Formula/dmd.rb

```diff
@@ -63,13 +58,15 @@ def install
     ]
 
     dmd_make_args = %W[
+      INSTALL_DIR=#{prefix}
       SYSCONFDIR=#{etc}
-      TARGET_CPU=X86
-      AUTO_BOOTSTRAP=1
+      HOST_DMD=#{Formula["ldc"].opt_bin/"ldmd2"}
       ENABLE_RELEASE=1
+      VERBOSE=1
     ]
 
-    system "make", *dmd_make_args, *make_args
+    system "ldc2", "src/build.d", "-of=src/build"
+    system "src/build", *dmd_make_args
 
     make_args.unshift "DMD_DIR=#{buildpath}", "DRUNTIME_PATH=#{buildpath}/druntime", "PHOBOS_PATH=#{buildpath}/phobos"
```

Comment:
You can add this directly to `make_args` since you're no longer using it in the `system` calls above.